tests/menubar-editor/data/menubar-edit-tests.csv

```diff
@@ -0,0 +1,28 @@
+Status,Test No,Title,Mode,Testing Task,specific_user_instruction(Task),output_assertion(Success Criteria),2,3,4,5,6
+Asking clarification for a group(1/30),1,Activating 'Text Formatting' menubar switches mode from reading to interaction,reading,activate menubar,activate the 'Text Formatting' menubar,the screen reader switches to mode that interaction,,,,,
+Asking clarification for a group(1/30),2,Navigating to 'Text Formatting' menubar switches mode from reading to interaction,reading,navigate to menubar with keys that switch modes,navigate to the 'Text Formatting' menubar,After command; JAWS is in Forms Mode,,,,,
+,3,Navigating to menubar,reading,navigate to menubar ,navigate to the first menuitem(Font) in menubar,The role 'menubar' is conveyed,The name 'Text Formatting' is spoken,The role 'menuitem' is conveyed,The name 'Font' is spoken,The state of the checkbox aria-haspopup is conveyed,The position in menubar and number of menubar items is spoken

```

Comment:
I haven't fully reviewed these CSV files, but I did notice "The state of the checkbox aria-haspopup is conveyed". The attribute `aria-haspopup` allows for many values. I don't think it is clear what exactly is expected here. IMO, to pass, the screen reader must convey that the menuitem opens a menu. As written, a tester might interpret this to mean any value associated with `aria-haspopup` may pass.

Additionally, I'm not sure what "checkbox" is referring to.